gitops/azure-devops/build.sh

```diff
@@ -70,11 +69,33 @@ function download_fab() {
     unzip fab-v$VERSION_TO_DOWNLOAD-$os-amd64.zip -d fab
 }
 
+# Install the HLD repo if it's not running as part of the HLD build pipeline
+function install_hld() {
+    echo "DOWNLOADING HLD REPO"
+    echo "git clone $HLD_PATH"
+    git clone $HLD_PATH
+    # Extract repo name from url
+    repo=${HLD_PATH##*/}
```

Comment:
@samiyaakhtar do we need to update our documentation for this new variable? Is it backwards compatible?

Reply:
This variable is not required (so yes, it's backwards compatible, I verified it as well) it's just for scenarios where the pipeline is kicked off externally. I will update documentation